.github/workflows/build-test.yml

```diff
@@ -765,15 +777,34 @@ jobs:
         command: install
         args: --git https://github.com/rustwasm/twiggy.git --rev ${{ steps.twiggy-version.outputs.hash }} twiggy
 
+    - name: Install npm dependencies from cache
+      uses: actions/cache@v2
+      id: npm-cache
+      with:
+        path: ffi/diplomat/wasm/node_modules
+        key: npm-${{ hashFiles('ffi/diplomat/wasm/package-lock.json') }}
```

Comment:
Suggestion: add `${{ runner.os }}` to the key, as in the examples at https://github.com/actions/cache, in case this starts being used on multiple OS in the future.

Reply:
Done.